src/routes/auth/login.ts

```diff
@@ -74,7 +74,7 @@ async function loginAccount({ body, headers }: Request, res: Response): Promise<
   const account = await selectAccount(body)
 
   if (!account) {
-    return res.boom.badRequest('Account does not exist.')
+    return res.boom.badRequest(typeof password !== 'undefined' ? 'Invalid email or password': 'Invalid email')
```

Comment:
This is a nice one-liner! 

But could we break it up to increase the code readability + make a comment that we show "Invalid email" for magic link?

Reply:
"Good code is code that can be understood by a junior engineer. Great code can be understood by a first year CS freshman. The best code is no code at all."

https://old.reddit.com/r/ExperiencedDevs/comments/nmodyl/drunk_post_things_ive_learned_as_a_sr_engineer/